templates/credentials/sign-up.html

```diff
@@ -9,6 +9,15 @@
 {% endblock meta_copydoc %}
 
 {% block content %}
+  <script>
+    function getSubmitButton() {
+      return document.getElementById('formSubmitButton');
+    }
+
+    function disableSubmit() {
+      getSubmitButton().disabled = true;
```

Comment:
There is no null checking here. If the ID on the button changes or is not in place JS will error.